packages/fx-core/src/plugins/resource/apim/log.ts

```diff
@@ -42,8 +42,8 @@ export class LogMessages {
 
   public static readonly resourceNotFound = (resourceType: IName, resourceId?: string): string =>
     !resourceId
-      ? `[${ProjectConstants.pluginDisplayName}] Failed to find resource ${resourceType.displayName} in Azure.`
-      : `[${ProjectConstants.pluginDisplayName}] Failed to find resource ${resourceType.displayName} '${resourceId} in Azure.`;
+      ? `[${ProjectConstants.pluginDisplayName}] Resource ${resourceType.displayName} does not exist in Azure. Try to create a new one.`
+      : `[${ProjectConstants.pluginDisplayName}] Resource ${resourceType.displayName} '${resourceId}' does not exist in Azure. Try to create a new one.`;
```

Comment:
If this message template is also used for AAD app, then 'in Azure' is not that appropriate here.